scripts/test-runner.sh

```diff
@@ -47,8 +47,13 @@ if [[ $OS_TYPE != "fedora" ]]; then
     $PKG_MAN -y install epel-release
 fi
 
+# TEMPORARY WORKAROUND
+# TODO: Remove when a fixed official beakerlib version is released
+yum -y install https://sumsal.cz/dev/beakerlib-1.11centos-99.fc23.noarch.rpm
```

Comment:
Following Dalibor's recommendation, I filed a new bug where the progress on this issue can be tracked: [BZ#1416014](https://bugzilla.redhat.com/show_bug.cgi?id=1416014).